### ISSUE TITLE
Invoking jq to access a value of more than 18 digits causes value to be truncated  

### ISSUE THREAD

**Issue by roygbyte** [stedolan/jq#2313]
**Describe the bug**

Values that are greater than 18 digits are not returned with their 19th, 20th, 21st (etc) digit. Instead, they are returned with zeros. 

**To Reproduce**
Note that the second line below (i.e.: the returned value) has zeroes for the last 4 digits. 
```
$ echo '{"id":123435798719873812037}' | jq '.id'
123435798719873810000
```

**Expected behavior**
I would expect all digits to be returns

**Environment (please complete the following information):**
OS: Ubuntu 20.04
version: jq-1.5-1-a5b5cbe

**Comment by pkoppstein** [comment]
@roygbyte - This is a well-known issue, (see e.g. https://github.com/stedolan/jq/issues/2182) -- so well-known in fact that it has already been addressed, though only after the release of jq 1.6.  

The relevant "commits" were made on Oct 22, 2019.  There are various installation options for upgrading, too numerous to mention here, but if in doubt, you might wish to review:

* https://github.com/stedolan/jq/wiki/Installation
* https://github.com/stedolan/jq/wiki/FAQ#installation

In the meantime, please "close" this issue.

**Comment by roygbyte** [comment]
Thanks for the quick response.
